src/Compilers/VisualBasic/Portable/Symbols/ArrayTypeSymbol.vb

```diff
@@ -317,6 +317,41 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return IsSameType(obj, TypeCompareKind.ConsiderEverything)
         End Function
 
+        Friend Overrides Function Equals(t2 As TypeSymbol, comparison As TypeCompareKind) As Boolean
```

Comment:
Specifically, types with custom modifiers. 

---
In reply to: [117133141](https://github.com/dotnet/roslyn/pull/19283#discussion_r117133141) [](ancestors = 117133141)